File: 39-prottyasha/TaskScoreBoard/TaskScoreBoard/Football.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TaskScoreBoard
{
    class Football : Sport
    { 
        public override void TeamInfo()
        {
            Console.WriteLine("Ravenclaw vs Slytherin\nT20\n");
        }

        public override void Scores()
        {
            Console.WriteLine("2 - 3\n");
        }

       
        public override void TopPlayers()
        {
            Console.WriteLine("Player of the match: Newt of Hufflepuff\n");
        }
    }
}

File: 36-Riyad/src/lab1/Football.java
package lab1;

public class Football extends Game{
	
	String teamName;
	String score;
	String bestPlayer;
	
	public Football(String teamName, String score, String bestPlayer) {
		this.teamName = teamName;
		this.score = score;
		this.bestPlayer = bestPlayer;
	}

	@Override
	void teamInfo() {
		System.out.println(teamName);
		System.out.println("== == ==");
		
		
		
	}

	@Override
	void score() {
		System.out.println(score);
		System.out.println("== == ==");
		
	}

	@Override
	void bestPlayer() {
		System.out.println("Player of the match: "+bestPlayer);
		
	}

}
